common/event_modifiers/wc_event_modifiers.txt

```diff
@@ -1628,4 +1628,12 @@ barrow_den_average_protection = {
 barrow_den_good_protection = {
 	icon = 5
 	defensive_plot_power_modifier = 0.5
+}
+ulduar_closed_down = {
```

Comment:
This modifier doesn't reduce revolt chance, so there may be revolt in Ulduar.

Reply:
fixed